### ISSUE TITLE
Config key `skip_output` not working

### ISSUE THREAD

**Issue by addiebarron** [evilmartians/lefthook#162]
Maybe I'm missing something integral here, but it seems like modular output options are being ignored for my setup. Details here:

`lefthook.yml` at project root: 
```
pre-commit:
  commands:
    commit-colors:
      run: commit-colors $(git rev-parse HEAD)
skip_output:
  - meta
```

sample output after `git commit` or `npx lefthook run pre-commit`:
```
Lefthook v0.7.2
RUNNING HOOKS GROUP: pre-commit

  EXECUTE > commit-colors

       
        Your commit color is Picton Blue
       
#55a9ea


SUMMARY: (done in 0.04 seconds)
✔️  commit-colors
```

The project in question is up-to-date over [here](https://github.com/addiebarron/bananacamp).

Expected behavior is that the version number and hook details would not be displayed. Thank you!!

**Comment by Maarius** [comment]
I have the same issue. The `skip_output` flag seems to be ignored.

**Comment by DmitryTsepelev** [comment]
@addiebarron @Maarius I've just tried to reproduce the issue on 0.7.3 and it seems to work fine, I do not see `Lefthook v0.7.2 RUNNING HOOKS GROUP: pre-commit` in the output. Could you please try again using the latest version?

**Comment by Maarius** [comment]
@DmitryTsepelev thanks for the mention. I can confirm that I cannot reproduce this issue on `0.7.3`. So looks like upgrading to latest version fixes it. Let's close if @addiebarron agrees.

**Comment by addiebarron** [comment]
Thanks all, I can confirm the issue is fixed on 0.7.3 -- appreciate your work. Closing now
